src/Tools/FormExporter.php

```diff
@@ -224,6 +246,7 @@ protected function extractDataRows(array $resultsArray)
     protected function extractColumnNames($dataRow)
     {
         $data = json_decode($dataRow[$this->dataFieldName], true);
+        $data['created'] = $dataRow['created'];
```

Comment:
Let's put a note in there about why that's hard-coded so that future me doesn't forget.